nipoppy/data/examples/sample_bids_dataset_description.json

```
@@ -0,0 +1,5 @@
+{
+    "Name": "CHANGEME",
```

FWIW, in heudiconv I have convention of annotating such locations with `TODO` so I could `git grep TODO`. So may be

```suggestion
    "Name": "TODO: CHANGEME",
```

but as @asmacdo suggested -- could be filled in with directory name.